src/pages/CoinPage/index.js

```diff
@@ -50,35 +41,68 @@ export default class CoinPage extends React.Component {
         )}
         {hasData && (
           <div>
-            <StyledRow>
-              <Col>
-                <Img src={coinData.image.large} />
-              </Col>
-              <Coin>
-                <Rank>#{coinData.marketCapRank}</Rank>
-                <Name>{coinData.name}</Name>
-                <Ticker>{coinData.symbol}</Ticker>
-                <Favorite>
-                  <FavoriteCoin />
-                </Favorite>
-              </Coin>
-            </StyledRow>
+            <CoinPageHeader
+              img={coinData.image.large}
+              rank={coinData.marketCapRank}
+              name={coinData.name}
+              ticker={coinData.symbol}
+              website={coinData.links.homepage[0]}
+              contractAddress={coinData.contractAddress}
+              categories={coinData.categories}
+              currentPrice={coinData.marketData.currentPrice.usd}
+              priceChange24h={coinData.marketData.priceChange24hInCurrency.usd}
+              priceChangePercentage24h={
+                coinData.marketData.priceChangePercentage24h
+              }
+              marketCap={coinData.marketData.marketCap.usd}
+              marketCapChangePercentage24h={
+                coinData.marketData.marketCapChangePercentage24h
+              }
+              fullyDilutedValuation={
+                coinData.marketData.fullyDilutedValuation.usd
+              }
+              totalVolume={coinData.marketData.totalVolume.usd}
+              circulatingSupply={coinData.marketData.circulatingSupply}
+              totalSupply={coinData.marketData.totalSupply}
```

Comment:
can you destructure some of this stuff? or pass the whole coin to coin page header instead of doing this?